server/src/main/java/org/jboss/as/server/deployment/Phase.java

```diff
@@ -234,6 +234,7 @@ public AttachmentKey<?> getPhaseKey() {
     public static final int STRUCTURE_GLOBAL_MODULES                    = 0x1E00;
     public static final int STRUCTURE_GLOBAL_DIRECTORIES                = 0x1E01;
     public static final int STRUCTURE_NAMING_EXTERNAL_CONTEXTS          = 0x1F00;
+    public static final int STRUCTURE_NAMING_JDK_DEPENDENCIES           = 0x1F01;
```

Comment:
This should probably be between the existing two constants rather than incrementing the prior constant by one as this prevents inserting another constant between STRUCTURE_NAMING_EXTERNAL_CONTEXT and STRUCTURE_NAMING_JDK_DEPENDENCIES if the need arose in the future.

Reply:
OK